### DISCUSSION TITLE
未来可实现功能

### DISCUSSION THREAD

**Discussion by Marcowky** [Marcowky/Web_Fengyang#41] [Ideas]
- 文章详情页的评论
- 后台页面
- 搜索无文章时的占位空间
- 登录的邮箱验证
- 将页面配置放数据库，使其可在后台修改(过多无意义操作，放弃)
- 轮播图下的一句话咨询
- 顶部栏动态效果
- 轮播图美化
- 全局背景图
- 删除用户后文章作者设为'用户已注销'

**Comment by Jiangt333** [comment]
- 页脚完善，“首页“改成”景区概况“

**Comment by Marcowky** [comment]
- 封装router
